modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/jaxrs/context/provider/ObjectDefinitionContextProvider.java

```diff
@@ -28,15 +32,30 @@
 public class ObjectDefinitionContextProvider
 	implements ContextProvider<ObjectDefinition> {
 
-	public ObjectDefinitionContextProvider(ObjectDefinition objectDefinition) {
-		_objectDefinition = objectDefinition;
+	public ObjectDefinitionContextProvider(
+		ObjectDefinitionRESTContextPathRegistry
+			objectDefinitionRESTContextPathRegistry,
+		Portal portal) {
+
+		_objectDefinitionRESTContextPathRegistry =			objectDefinitionRESTContextPathRegistry;
```

Comment:
Seems that there is some kind of error with the SourceFormatter. I need to check it before forwarding it